qadence/backends/pulser/config.py

```diff
@@ -49,6 +50,12 @@ class Configuration(BackendConfiguration):
     # interaction type
     interaction: Interaction = Interaction.NN
 
+    # platform to execute sequence on on the cloud
+    cloud_platform: Optional[EmulatorType] = None
+
+    # credentials for connecting to the cloud platform
+    cloud_credentials: Optional[dict] = None
```

Comment:
It is likely more elegant to define a dataclass for this and add also the `cloud_platform` attribute directly in the dataclass. In this way, the user needs to specify only one field and it is easier to document.